draft-ietf-openpgp-pqc.md

```diff
@@ -1016,10 +1026,12 @@ The algorithm-specific fields consists of:
    - The wrapped session key represented as an octet string, i.e., the output
      of the encryption procedure described in {{ecc-mlkem-encryption}}.
 
-Note that unlike most public-key algorithms, in the case of a v3 PKESK packet,
-the symmetric algorithm identifier is not encrypted.  Instead, it is prepended
-to the encrypted session key in plaintext.  In this case, the symmetric
-algorithm used MUST be AES-128, AES-192 or AES-256 (algorithm ID 7, 8 or 9).
+Note that unlike most public-key algorithms, for the ML-KEM composite schemes,
+in the case of a v3 PKESK packet, the symmetric algorithm identifier is not
+encrypted.  Instead, it is prepended to the encrypted session key in plaintext.
+In this case, the symmetric algorithm used MUST be AES-128, AES-192 or AES-256
+(algorithm ID 7, 8 or 9).
```

Comment:
Should we justify this?

I note that in crypto-refresh-13, the same concern is not explicitly justified in the text, but in commit 3eea8ee71215a2e1be24a3d45ac928aa098720a4 @twiss justifies it with:

> Since the symmetric algorithm ID is not encrypted nor authenticated in v3 PKESKs, mandate that AES is used, to prevent a (remote) risk of cross-algorithm attacks.

Reply:
I would even go so far to say that the chosen approach is not 100% waterproof. It prevents cross-algorithm attacks, but not necessarily cross-key-size attacks in all cases. If an implementation accepts a 256 bit value for an AES-128 key by dropping the second half, then such an attack is possible. I would clearly say that a secure and robust implemention should not accept oversized keys, but still: with the chosen approach we increase the attack surface without any justifiable reason, except that we want align with the crypto-refresh.

Reply:
In our implementations we already check the key size matching the algorithm, and error our otherwise. I see no issue in mandating this check.